src/server/express/handlers/jwt.ts

```diff
@@ -0,0 +1,18 @@
+import { Handler, Request } from 'express'
+import { sign } from '../../jwt'
+import log from '../../log'
+
+// eslint-disable-next-line unicorn/consistent-function-scoping
```

Comment:
**Nitpick**

Might we get an explanation for this?

Reply:
I'm debating if we should disable this for arrow functions. What do you think?

 see [unicorn/consistent-function-scoping](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/consistent-function-scoping.md)

I disabled this to allow for "currying"

e.g.: this is invalid according to this rule
```typescript
const f = () => () =>
```

however, it does tell you that your function could be moved to the upper scope if defined within a scope, and no parameters are used. So it's somehow useful.